scripts/xcm_transfers/xcm/graph/config_traversal.py

```diff
@@ -0,0 +1,49 @@
+from typing import Callable, List, Iterator
```

Comment:
no usage for Callable:

```suggestion
from typing import List, Iterator
```